### DISCUSSION TITLE
Edit code

### DISCUSSION THREAD

**Discussion by ghost** [community/community#12479] [Mobile] [answered]
I want to edit my code on the mobile App.

**Comment by stevepopovich** [comment]
HI! Thanks for the feedback, I will pass this along to the team.

**Comment by stevepopovich** [comment, marked as answer]
HI! This is shipped on iOS and coming soon on Android. Thanks!

https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code/

**Comment by Preetiraj3697** [comment]
Which Code editor?

**Reply by stevepopovich** [reply]
It is an internal code editor inside the GitHub Mobile app!
